client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -191,6 +191,18 @@ class Upload extends React.Component {
 	}
 }
 
+Upload.propTypes = {
```

Comment:
I think that we should use ES7 React static propTypes. Calypso uses this style all around.
https://babeljs.io/blog/2015/06/07/react-on-es6-plus